### PR TITLE
[Post] Barcode

### PR DESCRIPTION
**Pull request by Sqyphen**
**Description**
Adds a new blog post page describing the Barcode project

**Implementation**
Added html page and the images for the page
Also updates some gem packages so that the `bundle exec jekyll serve` runs

**Screenshot**
screenshot 2019-01-29 at 14 49 08

**.env changes**
n/a

**Additional requirements**
May need to run `bundle install` to make sure you have the latest packages

### CODE REVIEW

File: _posts/2019-01-29-barcode.html [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+
+<p>Pleasing to look at, however its ability to show a meaningful summary of the news is debatable.</p>
+
+<p>While building the above, and looking for image summarisation inspiration, we came across an image created from every single frame of the Harry Potter movie series.</p>
+
+{% include figure.html src="/assets/uploads/2019/01/barcode_hp_films.jpg" description="All the Harry Potter movie frames in one image <a href='https://www.reddit.com/r/dataisbeautiful/comments/194gmp/all_the_harry_potter_frames_from_all_the_movies/' target='_blank'>https://www.reddit.com/r/dataisbeautiful/comments/194gmp/all_the_harry_potter_frames_from_all_the_movies/</a> " width="80%" %}
+
+<p>Even though you cannot see the contents of each individual frame you can see the movies getting darker over each iteration and the (spoilers) white train station scene at the end of the series.</p>
+
+<p>So we made a version of the for the FT news and called it <b>Stretched Images:</b></p>
```
Comment (Lily2point0) [inline _posts/2019-01-29-barcode.html]: I don't think bold is necessary here

Comment (Sqyphen) [reply _posts/2019-01-29-barcode.html]: Cool, removed

File: _posts/2019-02-05-barcode.html [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+title:  "Barcode"
+date:   2019-02-05 09:30:30 +0100
+categories: Experiment
+teaser: Experiment
+excerpt: >
+  Creating art out of headline images
+---
+<p><a href="https://github.com/ftlabs/barcode" target="_blank">Barcode</a> is an internal service/endpoint which gathers all of the headline images used in FT articles from a provided date range, the last 24 hours for example, and squashes them to give one condensed image. The final result looks similar to a coloured barcode.</p>
+
+{% include figure.html src="/assets/uploads/2019/01/barcode_header_example.jpg" description="Parameters used to generate this image: width=1024, height=768, dateFrom=2018-11-13, timeFrom=00:00:00, dateTo=2018-11-14, timeTo=00:00:00, orientation=v, fit=fill, order=colour, sort=asc" width="80%" %}
```
Comment (railsagainstignorance) [inline _posts/2019-02-05-barcode.html]: I would say this is unneeded detail in the img description. Clearer would be "Barcode of articles published during 2018-11-13, sorted by colour"

Comment (Sqyphen) [reply _posts/2019-02-05-barcode.html]: Ok, amended

**Pull request merged**